assignment/checkpoints/terminal.txt

```diff
@@ -1,21 +1,22 @@
 # Type your responses on a new line
 
 # 1 - What is the command for listing the files in the current folder (directory)?
-
+    ls
 # 2 - What is the command for changing directories?
-
+    cd
 # 3 - What is the command for going "up" a folder (to a parent directory)?
-
+    ..
```

Comment:
Careful! While `..` refers to the parent directory, you would need `cd ..` to actually "go up".